tests/fixtures/course.json

```diff
@@ -1391,5 +1391,65 @@
 			}
 		],
 		"status_code": 200
+	},
+	"add_grading_standards": {
+		"method": "POST",
+		"endpoint": "courses/1/grading_standards",
+		"data": {
+			"title": "Grading Standard 1",
+			"id": 1,
+			"context_id": 1,
+			"context_type": "Course",
+			"grading_scheme": [
+			  {"name": "A", "value": 0.9},
+			  {"name": "B", "value": 0.8},
+			  {"name": "C", "value": 0.7}
+			]
+		  },
+		"status_code": 200
+	},
+	"get_grading_standards": {
+		"method": "GET",
+		"endpoint": "courses/1/grading_standards",
+		"data": [
+			{
+			"title": "Grading Standard 1",
+			"id": 2,
+			"context_id": 1,
+			"context_type": "Course",
+			"grading_scheme": [
+			  {"name": "A", "value": 0.9},
+			  {"name": "B", "value": 0.8},
+			  {"name": "C", "value": 0.7}
+			]
+		  },
+		  {
+			"title": "Grading Standard 2",
+			"id": 3,
+			"context_id": 1,
+			"context_type": "Course",
+			"grading_scheme": [
+			  {"name": "D", "value": 0.6},
+			  {"name": "F", "value": 0.5}
+			]
+		  }
+		],
+		"status_code": 200
+	},
+	"get_single_grading_standard": {
+		"method": "GET",
+		"endpoint": "courses/1/grading_standards/1",
+		"data": {
+			"title": "Grading Standard 1",
+			"id": 4,
```

Comment:
In this case, the request is to `courses/1/grading_standards/1` which means the grading standard ID should be 1, not 4.